src/app/dataset-view/additional-components/overview-component/overview-component.html

```diff
@@ -286,18 +286,51 @@ <h2 class="dataset-info--subtitle">
                     </div>
                     <div
                         class="dataset-info--block"
-                        *ngIf="currentState && currentState.overview.metadata.currentWatermark"
+                        *ngIf="currentState.overview.kind==='ROOT'"
```

Comment:
I am not sure this condition is valid. Why do you think this should be disallowed for derived datasets?